wandb/sdk/data_types/image.py

```diff
@@ -65,65 +65,72 @@ def _server_accepts_artifact_path() -> bool:
 class Image(BatchableMedia):
     """Format images for logging to W&B.
 
+    See https://pillow.readthedocs.io/en/stable/handbook/concepts.html#modes
+    for more information on modes.
+
     Args:
-        data_or_path: (numpy array, string, io) Accepts numpy array of
+        data_or_path: Accepts numpy array of
```

Comment:
Could you also reformat this block? Lines should be up to 80 characters long (but not longer).